docs/source/user/contribute.md

```diff
@@ -0,0 +1,14 @@
+# Contributions and Collaborations
+
+PySkinDose is open-source and available for download from the project repository. Contributions and
+collaborations for further development of PySkinDose is greatly appreciated. Listed below is a couple
+of examples:
+
+- New functions for more accurate patient skin dose estimation
+- Added calculation of uncertainties
+- Suggestions of further development
+- Data for implementation of new/other FGI devices
+- Recommendations of new literature about patient skin dose estimation
+- Alteration of computational phantoms
+- Reviews of user friendliness
+- Etc.
```

Comment:
Do we need this bullet-point when the sentence above says _Listed below are a couple of examples:_ ?

Reply:
Fair point. Removed "etc."